ANSWERS.md

```diff
@@ -0,0 +1,8 @@
+1) What are PropTypes used for?
+They are a way to keep track of and validate what data types should be passed down through which components.  
+2)What is a lifecycle event in React?
+It's a function that can be accessed at the stage between the mounting and unmounting of a component in order to trigger certain actions at specific times of an App and its component's lifecycles; things like API calls, and deactivating event listeners on unmount, etc.   
+3)What is a Higher Order Component? A higher order component is a component that takes another component as its props and can be used to modify the style of the component or execute some kind of logic sequence involving it. 
```

Comment:
the important concept is that it takes a component in and it returns a component (which does what you say it does in your answer)